api/v1alpha1/flowcollector_types.go

```diff
@@ -194,6 +207,19 @@ type FlowCollectorConsolePlugin struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
 }
 
+// CNO defines the desired configuration related to the Cluster Network Configuration
+type ClusterNetworkOperator struct {
+	// Important: Run "make generate" to regenerate code after modifying this file
+
+	//+kubebuilder:default:=ovs-flows-config
+	// Namespace  where the configmap is going to be deployed.
```

Comment:
btw (if we keep this field) the description is inaccurate

Reply:
🤦